frontend/router/paths.ts

```diff
@@ -1,6 +1,7 @@
 import { IPolicy } from "../interfaces/policy";
 import URL_PREFIX from "./url_prefix";
 
+// Note: changes to paths.ts should change page_titles.ts respectively
```

Comment:
Why don't we have the code enforce this without requiring extra work to update? We could do that various ways. For future iterations.